tests/jsTests/indexSpec.js

```diff
@@ -1,5 +1,9 @@
-describe("Tests for show error", function() {
-	var msg = 'this is a test message';
+/**
+ * Suite : Tests for showing error on incorrect field entry
+ * Description : This is a test suite for showing error in case of an invalid field entry
+ */
+describe("Tests for showing error on incorrect field entry", function() {
+	var msg = 'This is a test message for showing error';
```

Comment:
The text message can be anything as it plays a role setting div content to that message. Altering text message won't play a major role in the tests. It is kept generic as we are testing any field in the form on index.html. Making it specific gives the clue that it tests just single field. These specs are supposed to be written generically to ensure that the functionality (in this case, showing error) works same for any type of field. The function showError works generically and ensures an error message is displayed properly.

Eg. We might add a volunteer's address of correspondence. In that case, the functionality of showing error should be the same. The error div must show the error message which is given. Hence the msg here is generic.